src/SDKs/DataFactory/Management.DataFactory/Properties/AssemblyInfo.cs

```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Data Factory Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Factory Resources.")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
`AssemblyVersion` remains unchanged, `AssemblyFileVersion` must be changed

Reply:
A. If there are breaking changes, the major version must be bumped here (`2.0.0`)
B. I looked at the commit provided and there are some additive changes only. This should not be a breaking change, which means we still need to keep `AssemblyFileVersion` unchanged

Reply:
We follow the Guidelines for breaking changes that Abhijeet sent . There different types of breaking changes:
1. API breaking
2. Client breaking

In case of API breaking we bump the major version, in case of client breaking, we bump the minor version, in case of just new additional types, we update the 3rd number. In first two cases we update both assembly file version and assembly version. We have been doing the same all this time. The constructor signature in the last commit has changed, which can result in breaking of the client code, hence the minor version bump.